client/tsconfig.json

```diff
@@ -1,23 +1,13 @@
 {
-	"compilerOptions": {
-		"target": "es6",
-		"lib": [ "es6" ],
-		"module": "commonjs",
-		"outDir": "out",
-		"sourceMap": true,
-		"noLib": false,
-		"forceConsistentCasingInFileNames": true,
-		"noImplicitAny": true,
-		"noImplicitReturns": true,
-		"noFallthroughCasesInSwitch": true,
```

Comment:
There are still a few flags that are gone, right? Like `noFallthroughCasesInSwitch`, `noUnusedLocals`.

Reply:
Ah no sorry, I see they were moved to the toplevel file.